### ISSUE TITLE
Error Deploying Code

### ISSUE THREAD

**Issue by bmodahl** [forcedotcom/cli#930]
This seems to happen on random orgs I have already authorized via VS Code and any new org I try to authorize via VS Code

Steps to reproduce:
1. Use the standard sfdx force:source:deploy command
2. One thing to note is that after 5-10 consecutive save attempts I am able to push Apex Classes through. I have not had a successful push with LWC

Results:
`SfdxError: 
<html>
<body>

<table cellspacing=10>
    <tr>
        <td><span style="font-weight: bold; font-size: 12pt;">Illegal Request</span></td>
    </tr>
    <tr>
        <td>You have sent us an Illegal URL or an improperly formatted request. 
        </td>
    </tr>
    <tr>
        <td>
            
        </td>
    </tr>
</table>

<script type="text/javascript">function bodyOnLoad(){if(window.PreferenceBits){window.PreferenceBits.prototype.csrfToken="null";};}function bodyOnBeforeUnload(){}function bodyOnFocus(){}function bodyOnUnload(){}</script>
			
</body>
</html>

`

**Comment by firatesmer** [comment]
Hi,
I have the same issue. I can't deploy my code into org via Salesforce CLI.

CLI Version: sfdx-cli/7.90.2-b8f9206a5c win32-x64 node-v14.15.4
VS Code version: v.1.54.3
Windows 10 20H2

**Comment by shetzel** [comment]
Duplicate of #860 
Please see details in that thread for workarounds.  Switching from REST to SOAP should unblock you.
